app/controllers/posts_controller.rb

```diff
@@ -1,10 +1,15 @@
 class PostsController < ApplicationController
   def index
-    @posts = Post.all
+    user_id = params[:user_id]
+    @user = User.find(user_id)
+    @posts = Post.where(author_id: user_id)
   end
 
-  # GET /users/:user_id/posts/:id
   def show
-    params[:id]
+    user_id = params[:user_id]
+    post_id = params[:id]
+    @user = User.find(user_id)
+    @post = Post.find(post_id)
+    @comments = Comment.where(post_id)
   end
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/216752721-aa8e20ba-9861-4430-8922-1c997a8df6c2.png)

**[Optional]**

- [ ] When i tried to get an individual post then this error popped up on my screen :fearful: Kindly fix it :+1: